app/views/world/index.html.erb

```diff
@@ -46,6 +46,37 @@
       </ol>
     </div>
   </section>
+
+  <section id="<%= I18n.t("world_locations.type.international_delegation", count: 2).parameterize %>">
+    <div class="govuk-grid-column-one-third">
+      <header class="type-heading">
+        <h2 class="world-locations-heading"><%= I18n.t("world_locations.type.international_delegation", count: 2) %></h2>
+        <p class="govuk-visually-hidden"><%= t('world_locations.count_prefix') %>
+          <span class="js-accessible-item-count"><%= @presented_index.international_delegations_count %></span> <%= I18n.t("world_locations.type.international_delegation", count: 2) %>
```

Comment:
Style comment

This is just a style thing, so feel free to ignore. But if we just have `@presented_index.international_delegation.count` here instead, we could lose `delegate :count, to: :international_delegations, prefix: true` from the presenter. Likewise for `world_locations_count`. Two lines of code replaced by two `.`'s :-) ... I think??

Reply:
I'm going to leave this because we have have to make `world_locations` public and I like that it's private because its more clear to any future developers that its not used in the view